app/javascript/components/widgets/messages/message.jsx

```diff
@@ -27,42 +30,37 @@ const Column = styled.div`
   flex-direction: column;
 `;
 
-const Message = ({ width, height, message, imageWidth, imageHeight }) => {
-  const widthFloat = parseFloat(width);
-  const heightFloat = parseFloat(height);
+const SunMoon = styled.img`
+  width: 100px;
+  height: 100px;
+  align-self: center;
+`;
 
-  const svgWidth = width;
-  const svgHeight = height;
+const Message = ({ showImage, message }) => {
   return (
-    <Column>
-      <svg width={svgWidth} height={svgHeight}>
-        <image
-          x={((widthFloat - parseFloat(imageWidth)) / 2).toString()}
-          y={(heightFloat - parseFloat(imageHeight) + 1).toString()}
-          width={imageWidth}
-          height={imageHeight}
-          xlinkHref={sunMoon}
-          position="absolute"
-        />
-      </svg>
-      <MessageDisplay>{message}</MessageDisplay>
-    </Column>
+    <>
+      {showImage && (
+        <Column>
+          {showImage && <SunMoon src={sunMoon} />}
+          <MessageDisplay>{message}</MessageDisplay>
+        </Column>
+      )}
+      {!showImage && (
+        <Column>
+          <MessageDisplay>{message}</MessageDisplay>
+        </Column>
+      )}
+    </>
```

Comment:
This seems odd. Couldn't we just use `{showImage && <SunMoon src={sunMoon} />}` and not need the outer `showImage &&` or containing fragment

Reply:
Ah ok. `Column` eventually changes. Is image the only difference between these two or is the split SideBar message vs Panel message? If its the latter, I think it would be better to just name two separate components that are exported.